docs/source/zh-cn/core/httpclient.md

```diff
@@ -667,7 +667,7 @@ console.log(result.res.timing);
 先安装和开启插件：
 
 ```bash
-$ npm i egg-development-proxyagent --save
+$ npm i egg-development-proxyagent --save-dev
```

Comment:
if mount at `config/plugin.js`, must be `dep` not `devdep`

Reply:
Thanks @atian25 , but I am referring to [here](https://github.com/eggjs/egg-development-proxyagent/blob/master/README.md). What is the difference.